### ISSUE TITLE
mojaloop/helm - HUB_PARTICIPANT.NAME in default.json is not consistent across services

### ISSUE THREAD

**Issue by shashi165** [mojaloop/project#2243]
**Summary**:
In mojaloop/helm - at some places in `default.json` `HUB_PARTICIPANT.NAME ='hub'` is all lowercase, needs to be 'Hub'
Since in central-ledger it's 'Hub' and the participant is created with that value.  Later on any service that tries to get details of Hub also needs to use 'Hub' as participant name.
Services like `central-event-processor` and `email-notifier` have 'hub' as participant name and hence they are failing with below error
```
2021-05-31T10:02:42.981Z - info: DfspNotificationEndpoints failed with error: StatusCodeError: 500 - {"errorInformation":{"errorCode":"2001","errorDescription":"Internal server error - Participant does not exist"}}
```

**Severity**:
Medium 

**Priority**:
Critical - Reason - Mowali production emails are not being sent out since central-event-processor is failing

**Expected Behavior**
In all the default.json the HUB_PARTICIPANT.NAME should have a consistent value

**Steps to Reproduce**
  1.
  2.
  3.

**Specifications**
  - Component (if known): mojaloop/helm
  - Version: latest
  - Platform: mojaloop
  - Subsystem:
  - Type of testing:
  - Bug found/raised by: @shashi165 

**Notes**:

- Severity when opened: Medium
- Priority when opened: Critical

**Comment by mdebarros** [comment]
The issue is being addressed as part of Helm v12.1.0 release -> https://github.com/mojaloop/helm/pull/436

**Comment by elnyry-sam-k** [comment]
@shashi165 should be fixed with https://github.com/mojaloop/helm/releases/tag/v13.0.0, please confirm when you get a chance.. (if there's an issue, please reopen)
